CHANGES/977.bugfix

```diff
@@ -0,0 +1,4 @@
+Introduced a repair command for fixing the media types of manifests that could be incorrectly
```

Comment:
Introduced a repair command to fix incorrectly stored manifest media_type in the db as a result of sync task.

Reply:
"... that could have been ..." The moment this code is installed it should no longer be possible.